.travis.yml

```diff
@@ -2,8 +2,14 @@ language: python
 python:
   - "2.7"
   - "3.4"
+env:
+  - DJANGO_VERSION=1.7
+  - DJANGO_VERSION=1.8
+  - DJANGO_VERSION=1.9b1
 install:
   - pip install -r dev-requirements.txt
+  - pip uninstall django --yes
```

Comment:
Rather than install and uninstall, can we just create a separate test-requirements.txt without Django in it and use that?